capabilities/translation/analyze-plain-text-sentiment/maps/google-apis-language.suma

```diff
@@ -0,0 +1,72 @@
+profile = "translation/analyze-plain-text-sentiment@1.0"
+provider = "google-apis-language"
+
+map AnalyzePlainTextSentiment {
+
+  http POST "/v1/documents:analyzeSentiment" {
```

Comment:
they really have urls with colon ? 🤯

Reply:
yes see the docs: https://cloud.google.com/natural-language/docs/reference/rest/v1/documents/analyzeSentiment